src/main.rs

```diff
@@ -138,8 +138,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
 
     println!("Loading recipes");
-    let api_recipes: Vec<api::Recipe> =
-        request::ensure_paginated_cache(&api_recipes_path, "recipes").await?;
+    let api_recipes = {
+        let mut api_recipes: Vec<api::Recipe> = request::ensure_paginated_cache(
+            &api_recipes_path, "recipes"
+        ).await?;
+        // Blacklist a couple of known bad recipes
+        for id in vec![2812, 2825] {
```

Comment:
The vec could be replaced with a constant, e.g:
```rust
// ignore inaccurate recipes: https://github.com/gw2efficiency/issues/issues/1532
const RECIPE_BACKLIST_IDS: &[u32] = &[
    2812,  // Minor Rune of the Air
    2825   // Major Rune of the Air
];
```

Reply:
Yeah... I knew I shouldn't hardcode it... :D. I'll do it right.

I was considering adding them as a tagalong file, but:
- there should be few enough that it's fine to put them in the source
- it should be infrequent to add them, so it's fine to wait for a new build to blacklist them
- tagalong files are annoying

Reply:
Fixed